core/src/main/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolver.java

```diff
@@ -693,31 +694,43 @@ private static IndexNotFoundException infe(String expression) {
             return excludeState;
         }
 
-        private static Map<String, AliasOrIndex> matches(MetaData metaData, String expression) {
+        public static Map<String, AliasOrIndex> matches(Context context, MetaData metaData, String expression) {
```

Comment:
I had to change it to public for the unit test : `WildcardExpressionResolverTests#testConcreteIndicesWildcardAndAliases` 

Of course I can remove the test and revert to `private`